backend/EkoFunkcje/EkoFunkcje/Features/Comments/AddCommentFunction.cs

```diff
@@ -1,30 +1,27 @@
-using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
-using System.IO;
 using System.Threading.Tasks;
+using AzureFunctions.Extensions.Swashbuckle.Attribute;
 using EkoFunkcje.Models;
 using EkoFunkcje.Models.Dto;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
-using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Table;
 using Newtonsoft.Json;
 
-namespace EkoFunkcje
+namespace EkoFunkcje.Features.Comments
 {
     public static class AddCommentFunction
     {
-        [FunctionName("AddCommentFunction")]
+        [FunctionName("AddComment")]
         public static async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] AddCommentDto commentDto,
-            [Table(Config.CommentsTableName, Connection = Config.StorageConnectionName)]CloudTable cloudTable,
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "AddComment")]
```

Comment:
Jeśli miałoby być RESTowo, to komentarze jako pod-zasób należny do zasobu interwencji, mogłyby by być dodawane pod takim adresem:
Metoda HTTP: 'post'
`Route="interventions/{interventionId}/comments"`

W ten sposób id interwencji byloby brane bezpośrednio z URL, a nie z CommentDto.

Reply:
Poprawione